backends/include/dppl_sycl_queue_interface.h

```diff
@@ -63,4 +63,16 @@ DPPLQueue_GetDevice (__dppl_keep const DPPLSyclQueueRef QRef);
 DPPL_API
 void DPPLQueue_Delete (__dppl_take DPPLSyclQueueRef QRef);
 
+/*!
+ * @brief C-API wrapper for sycl::queue::memcpy. It waits an event.
+ *
+ * @param    QRef           An opaque pointer to the sycl queue.
+ * @param    Dest           An USM pointer to the destination memory.
+ * @param    Src            An USM pointer to the source memory.
+ * @param    Count          A number of bytes to copy.
+ */
+DPPL_API
+void DPPLQueue_memcpy (__dppl_keep const DPPLSyclQueueRef QRef,
```

Comment:
Minor nitpick. Does it make sense to rename as DPPLQueue_Memcpy to be consistent with rest of the API?

Reply:
Done.